portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Sherry's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body>
+    <header id="gallery-head" class="gallery-background">
```

Comment:
Is this id used anywhere?

Reply:
I forgot to, but I just added it to the href for "Gallery" in navbar so if you're already in gallery.html and you want to go to head of the page it doesn't need a full refresh. `gallery.html#gallery-head`